packages/react-console/src/components/VncConsole/VncConsole.tsx

```diff
@@ -84,7 +84,7 @@ export class VncConsole extends React.Component<VncConsoleProps, VncConsoleState
     textDisconnected: 'Disconnected',
     textDisconnect: 'Disconnect'
   };
-  private rfb: RFB;
```

Comment:
ah thanks :) I was not sure how to fix this, and ts-ignore stopped working since last week!